frontend/projects/upgrade/src/app/core/experiments/experiments.service.ts

```diff
@@ -119,11 +119,11 @@ export class ExperimentService {
     );
   }
 
```

Comment:
can this be removed instead of commented out?

Reply:
Removed this part